packages/server/src/fhir/operations/utils/cms1500pdf.ts

```diff
@@ -0,0 +1,410 @@
+import {
+  formatAddress,
+  formatCodeableConcept,
+  formatMoney,
+  formatQuantity,
+  getDisplayString,
+  HTTP_HL7_ORG,
+} from '@medplum/core';
+import { Address, Claim, HumanName, Practitioner, RelatedPerson } from '@medplum/fhirtypes';
+import fs from 'node:fs';
+import path from 'node:path';
+import { Content, TDocumentDefinitions } from 'pdfmake/interfaces';
+import { getAuthenticatedContext } from '../../../context';
+
+const PAGE_WIDTH = 612;
+const PAGE_HEIGHT = 792;
+
+const insuranceTypes = new Set<string>([
+  'MEDICARE',
+  'MEDICAID',
+  'TRICARE',
+  'CHAMPVA',
+  'GROUP HEALTH PLAN',
+  'FECA BLK LUNG',
+  'OTHER',
+]);
+
+const CMS_1500_IMG_DATA_URL =
```

Comment:
We can avoid this.

See: https://pdfmake.github.io/docs/0.1/document-definition-object/images/

We can refer to the static file inline